mediacapture-streams/MediaStreamTrackEvent-constructor.https.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<title>MediaStreamTrackEvent constructor</title>
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#mediastreamtrackevent">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(function() {
+  assert_equals(MediaStreamTrackEvent.length, 2);
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type");
+  });
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", null);
+  });
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", undefined);
+  });
+}, "The eventInitDict argument is required");
+
+test(function() {
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", {});
+  });
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", { track: null });
+  });
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", { track: undefined });
+  });
+}, "The eventInitDict's track member is required.");
+
```

Comment:
Unfortunately not, "TODO: test constructors if defined."
https://github.com/w3c/testharness.js/blob/484fd841a6fcec5dd00efbc7d420bbd9e7dcbd89/idlharness.js#L799